StatisticsTest fails on Windows

_master_ (commit c27bb3863a20813e5e3d862d258ca9c3decedc19)

Failed on windows for all NodeJS jobs, e.g.: https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/NodeJS-10-windows/60/

```
Statistics with default period should contain statistics content:
       AssertionError: expected 0 to be above 0
       + expected - actual
       
       at test\statistics\StatisticsTest.js:101:80
```

--- a/test/statistics/StatisticsTest.js
+++ b/test/statistics/StatisticsTest.js
@@ -137,7 +137,7 @@ describe('StatisticsTest (default period)', function () {
             expect(extractStringStatValue(stats, 'os.maxFileDescriptorCount')).to.equal('');
             expect(extractStringStatValue(stats, 'os.openFileDescriptorCount')).to.equal('');
             expect(extractIntStatValue(stats, 'os.processCpuTime')).to.greaterThan(1000);
-            expect(extractFloatStatValue(stats, 'os.systemLoadAverage')).to.be.greaterThan(0);
+            expect(extractFloatStatValue(stats, 'os.systemLoadAverage')).to.be.at.least(0);
             expect(extractIntStatValue(stats, 'os.totalPhysicalMemorySize')).to.equal(os.totalmem());
             expect(extractStringStatValue(stats, 'os.totalSwapSpaceSize')).to.equal('');
             expect(extractIntStatValue(stats, 'runtime.availableProcessors')).to.equal(os.cpus().length);

